ksqldb-functional-tests/src/test/resources/query-validation-tests/elements.json

```diff
@@ -1184,7 +1184,7 @@
       ],
       "expectedException": {
         "type": "io.confluent.ksql.util.KsqlException",
-        "message": "Tables require a PRIMARY KEY. Please define the PRIMARY KEY.\nUse a partial schema to define the primary key and still load the value columns from the Schema Registry, for example:\n\tCREATE TABLE INPUT (ID INT PRIMARY KEY) WITH (...);"
```

Comment:
Why do we change this error message?

Reply:
In https://github.com/confluentinc/ksql/blob/master/ksqldb-engine/src/main/java/io/confluent/ksql/ddl/commands/CreateSourceFactory.java#L176, because we don't populate schema id unless explicitly provided user anymore.